src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -445,6 +451,7 @@ public static void Verify_InvalidUseUnsafeAccessor()
         Assert.Throws<BadImageFormatException>(() => InvalidCtorType());
         Assert.Throws<BadImageFormatException>(() => LookUpFailsOnPointers(null));
         Assert.Throws<BadImageFormatException>(() => LookUpFailsOnFunctionPointers(null));
+        Assert.Throws<BadImageFormatException>(() => new Invalid().NonStatic(string.Empty));
```

Comment:
Do we also throw for generic `UnsafeAccessor` methods, and `UnsafeAccessor` methods in generic types? I do not see the test for it.

Reply:
No, we do not. I just verified the lookup fails. I will add tests.